### ISSUE TITLE
feat(config): Add keys to be scanned to Zebra config file 

### ISSUE THREAD

**Issue by oxarbitrage** [ZcashFoundation/zebra#7941]
We want to add an array of keys to be scanned to the zebra config so we can add them to the key storage at startup. 

The keys field can be a vector of tuples  `Vec<(String, Option<height>)>` where the first argument is the viewing key as a string and height is an optional birth date. Or it can just be `Vec<String>` if we want to add the birth height later. 

Tasks:

- [ ] Add key list to zebra config. (New "scan" section)

**Comment by teor2345** [comment]
> This is blocked on #7904 as the keys present in the config should be added to the storage at startup.

If we need to unblock this ticket, we can split connecting the keys up to storage into another PR or ticket. Totally up to you.

> Tasks:
> 
> * [ ]  Add key list to zebra config. (New "scanner" section?)

I'd suggest `scan`, because most of our [config names match the crate names](https://github.com/ZcashFoundation/zebra/blob/main/zebrad/tests/common/configs/v1.4.0.toml)

**Comment by oxarbitrage** [comment]
I think you are right, i unblocked it, created https://github.com/ZcashFoundation/zebra/issues/7946 which is blocked on this one and in #7904
